airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml

```diff
@@ -1,9 +1,12 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
 connector_image: airbyte/source-open-exchange-rates:dev
-
 acceptance_tests:
   spec:
     tests:
       - spec_path: "source_open_exchange_rates/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: 0.1.0
```

Comment:
If the previous version isn't backward compatible you need to release a major version, change to 1.0.0 as a breaking change. Why are you update the schema and specification of this connector?